cdap-authorization-dataset-plugin/src/main/java/co/cask/cdap/security/authorization/DatasetBasedAuthorizer.java

```diff
@@ -67,11 +71,22 @@ public ACLDataset get() {
         return new ACLDataset(table);
       }
     };
+    Properties properties = context.getExtensionProperties();
+    if (properties.containsKey("superusers")) {
```

Comment:
Is it necessary that there should be at least one super user? If not, how will other users get access to different entities in CDAP?